chain/stmgr/call.go

```diff
@@ -133,9 +133,7 @@ func (sm *StateManager) Call(ctx context.Context, msg *types.Message, ts *types.
 
 	// TODO: maybe just use the invoker directly?
 	ret, err := vmi.ApplyImplicitMessage(ctx, msg)
-	if err != nil {
-		return nil, xerrors.Errorf("apply message failed: %w", err)
-	}
+	// Don't check for error here, we want to bubble it up, but also return ret as an InvocResult
```

Comment:
`ret` may be nil？ @geoff-vball

Reply:
@simlecode you're right. I think this PR has the right behaviour https://github.com/filecoin-project/lotus/pull/9582